RealFlight/Released_Models/Planes/DG-800_Glider/README.md

```diff
@@ -0,0 +1,41 @@
+DG-800 Glider
+
+Aircraft Model: DG-800-A
+
+Original Aircraft - Design: DG Flugzeugbau GmbH
+
+3D - Model: maxkop
+
+Physical Aircraft Model: maxkop, opjose
+
+Wing Span....... 13.12 (ft)
+Length............5.12 (ft)
+Weight............10.35 (lbs)
+Power............. Kontronik Fun 600-18 (12 V)
+Prop.............. Standard Folding Prop
+Battery.......... 12 Cell - 2100 mAh - Li - Poly
+
+
+RC Functions:
+Output1		Right inner dspoiler
+Output2		Elevator
+Output3		Throttle
+Output4		Rudder
+Output5		Left inner dspoiler
+Output6		Landing Gear
+Output7		Motor Deploy
+Output8		Wheel Brake
+Output9		Right outer dspoiler
+Output10	Left outer dspoiler
+Output11	AirBrake
+
+Parameter file/setup notes:
+
+AETR normal control order, full span aileron control
+RC5 is flaps, full throw progressive crow
+RC6 is landing gear, automatic altitude control enabled
+RC7 is arm/disarm and motor deploy
+RC8 is mode, interlink controller modes: MANUAL,FBWB,CRUISE
+RC9 is wheel brake
+
+tested in RealFlight9 using ArduPlane 4.0.5 stable in Sim_vehcile.py
```

Comment:
and it probably wasn't tested in SITL on 4.0.5 stable

Reply:
updated